### DISCUSSION TITLE
Adding Search function for Furniture for blacksmithing, woodworking, clothing, jeweler

### DISCUSSION THREAD

**Discussion by tbagwell** [Rhyono/CraftStore#44] [Ideas]
I would love to see if there could be a search function For the other crafting tables.. I waste so much time trying to find what I am trying to make for furniture. Love as how it can be searched on runes and cooking.. I think it would be something that could help a lot of players.. I never know what it happen to be listed under.

Thanks.

**Comment by tbagwell** [comment]
Also Alchemy also.

**Comment by bhakiagjunfikorty** [comment]
Introducing an enhanced search function, #44 now facilitates seamless exploration of a diverse range of furniture tailored for blacksmithing, woodworking, clothing, and jewelry crafting. This user-friendly feature optimizes the shopping experience, allowing enthusiasts to efficiently locate specialized furniture that aligns with their craft. Whether you're a blacksmith in need of sturdy workbenches or a jeweler seeking ergonomic seating, our improved search function ensures that your quest for the perfect furniture is not only simplified but also tailored to your specific artisanal requirements[.](https://thekingsfurniture.com/) Explore the extensive collection and elevate your workspace with precision and style.

**Comment by tbagwell** [comment]
Thank you for working on this idea.   I will check it out when you come live.   Or do you have a test build I can look at it to help test it for you.
Does this also cover Alchemy also?

Kept up the good work
